external/js/kyber/test/group/nist/index.js

```diff
@@ -0,0 +1,755 @@
+const kyber = require("../../../index.js");
```

Comment:
We should see about creating a set of test vectors using a reference impl of p256 (maybe boringSSL would be a nice choice), and then ensuring that both Kyber-go and kyber-js calculate all of them correctly.